purchases/src/main/kotlin/com/revenuecat/purchases/models/GoogleProrationMode.kt

```diff
@@ -60,6 +60,14 @@ enum class GoogleProrationMode(
      * On May 1st, Samwise is charged $36 for his new subscription tier and another $36 on May 1 of each year following.
      */
     IMMEDIATE_AND_CHARGE_PRORATED_PRICE(BillingFlowParams.ProrationMode.IMMEDIATE_AND_CHARGE_PRORATED_PRICE),
+
+    /**
+     * Replacement takes effect when the old plan expires, and the new price will be charged at the same time.
+     *
+     * Example: Samwise's Tier 1 subscription continues until it expires on April 30. On May 1st, the
+     * Tier 2 subscription takes effect, and Samwise is charged $36 for his new subscription tier.
+     */
+    DEFERRED(BillingFlowParams.ProrationMode.DEFERRED),
```

Comment:
this will trigger some "switch is not exhaustive" but I doubt people are switching on proration modes, so it should be ok and better to offer `DEFERRED` as an option